src/Teams/Data/ProgramTestRepos/ProgramTestRepository.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Teams.Domain;
+
+namespace Teams.Data.ProgramTestRepos
+{
+    public class ProgramTestRepository : IProgramTestRepository
+    {
+        private readonly ApplicationDbContext _context;
+        public ProgramTestRepository(ApplicationDbContext context)
+        {
+            _context = context;
+        }
+        public string Delete(int programCodeTestId)
+        {
+            var progTest = _context.ProgramTests.FirstOrDefault(x => x.Id == programCodeTestId);
+            if (progTest != null)
+            {
+                _context.ProgramTests.Remove(progTest);
+                _context.SaveChanges();
+            }
+            return "Deleted";
```

Comment:
Try to avoid using raw strings. Make a constant in the header of the class.
Having the constants in one place helps especially if the constants need to be changed.